cmd/ocm/tunnel/cmd.go

```diff
@@ -100,6 +107,11 @@ func run(cmd *cobra.Command, argv []string) error {
 	}
 	sshuttleArgs = append(sshuttleArgs, argv[1:]...)
 
+
+	if args.short {
+		path = "sshuttle"
```

Comment:
The resulting command should still work because `LookPath("sshuttle")` found it on PATH.

Q: do you need both modes?  Sounds like you could always print short `sshuttle ...`.
But I never use this, I don't have an opinion here.

@cblecker @drewandersonnz please review.

Reply:
+1 to requiring `sshuttle` in PATH. 

I agree, I don't think a `args.short` should be necessary, I feel like we should rely on path lookup to support multiple OS's.

I believe the intention behind this is because some people use `ocm` via https://github.com/openshift/ocm-container (a Fedora linux environment in a container), yet need to use `sshuttle` on macos, which is found in a different path. This makes copying and pasting the resulting command (which is printed a few lines lower) just that little bit more difficult. 

So to finish, +1 to removing the `short` flag and simply have `path = "sshuttle"`.

You could also just change the print: 
```
	// Output sshuttle command execution string for review
	fmt.Printf("\n# %s %s\n\n", "sshuttle", strings.Join(sshuttleArgs, " "))
```

Reply:
I just didn't want to remove the functionality completely.. but if you are ok with it, so am I.